features/custom_parameter.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Custom Parameter
```

Comment:
Actually looks like this was done in one of the following PRs so this can be ignored.